packages/oui-popover/src/popover.controller.js

```diff
@@ -119,6 +138,10 @@ export default class PopoverController {
 
         // Support for attribute `oui-popover`
         this.$element.attr("aria-expanded", true);
+
+        if (angular.isDefined(this.onOpen)) {
```

Comment:
Since the binding of events is '&', it will always be defined, with angular.noop().
The check here has little use.